src/tests/Loader/classloader/regressions/529206/vsw529206StaticCctor.cs

```diff
@@ -1,85 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-// regression test for VSWhidbey 529206 for throwing TypeInitialization inside class .cctor
```

Comment:
I am dubious of the value of this test. Not saying we should remove it but it is very odd.